jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -185,19 +183,44 @@ public static void checkOwnerAccessOnly(String filePath) throws IOException {
 
 	/**
 	 * Open the semaphore, post to it, and close it
+	 * @param ctrlDir directory containing the semaphore
+	 * @param semaphoreName name of the semaphore
+	 * @param numberOfTargets number of times to post to the semaphore
+	 * @param global open the semaphore in the global namespace (Windows only)
+	 * @return 0 on success
+	 */
+	native static int notifyVm(String ctrlDir, String semaphoreName,
+			int numberOfTargets, boolean global);
+
+	/**
+	 * Open the semaphore, post to it, and close it
+	 * @param ctrlDir directory containing the semaphore
+	 * @param semaphoreName name of the semaphore
 	 * @param numberOfTargets number of times to post to the semaphore
 	 * @return 0 on success
 	 */
-	native static int notifyVm(String ctrlDir, String SemaphoreName,
-			int numberOfTargets);
+	static int notifyVm(String ctrlDir, String semaphoreName, int numberOfTargets) {
```

Comment:
It seems unnecessary to add new API and keep the old API, just to avoid changing a few callers. By my count there are only 1 place that need to change.

Reply:
Fixed.